apps/cli/src/commands/run.command.ts

```diff
@@ -85,6 +85,9 @@ export default class RunCommand extends BaseCommand {
     Logger.info('Connecting to socket...')
     const host = this.baseUrl.substring(this.baseUrl.lastIndexOf('/') + 1)
     const websocketUrl = `${this.getWebsocketType(this.baseUrl)}://${host}/change-notifier`
+    const privateKey = (data as any).privateKey;
```

Comment:
I think, for this you can add something like this in `run.types.d.ts`:
![image](https://github.com/user-attachments/assets/fca6a576-7bde-46c1-a670-4db362659076)

Then, you can use this type in here:
![image](https://github.com/user-attachments/assets/e2a738ce-0be1-47cd-a9d5-50f73a6e3a23)
![image](https://github.com/user-attachments/assets/864292a3-c895-4b82-8f91-a9a5422ec52c)

PS, I'm sorry about the initial confusion. I overlooked that `ProjectRootConfig` didn't have the private key field.

Reply:
Done